app/models/ExportsDeclaration.scala

```diff
@@ -46,13 +46,17 @@ case class ExportsDeclaration(
   natureOfTransaction: Option[NatureOfTransaction] = None,
   seals: Seq[Seal] = Seq.empty
 ) {
-  def itemBy(itemId: String): Option[ExportItem] = items.find(_.id.equalsIgnoreCase(itemId))
-
   def updatedItem(itemId: String, update: ExportItem => ExportItem): ExportsDeclaration =
     itemBy(itemId).fold(this) { item =>
       val updated = update(item)
       copy(items = items.filterNot(_.id.equalsIgnoreCase(itemId)) + updated)
     }
+
+  def itemBy(itemId: String): Option[ExportItem] = items.find(_.id.equalsIgnoreCase(itemId))
+
+  def lrn(): Option[String] = this.consignmentReferences.map { _.lrn }
```

Comment:
For one line map better is use normal parenthesis instead of curly

Reply:
addressed